README.md

```diff
@@ -1,10 +1,29 @@
-# LMA light bundle
-This juju bundle deploys a light LMA stack with the following interrelated charms:
-- [grafana](https://charmhub.io/grafana-k8s) ([source](https://github.com/canonical/grafana-operator))
-- [prometheus](https://charmhub.io/prometheus-k8s) ([source](https://github.com/canonical/prometheus-operator))
-- [alertmanager](https://charmhub.io/alertmanager-k8s) ([source](https://github.com/canonical/alertmanager-operator))
+# LMA Light bundle
+
+At Canonical, LMA stands for "Logs, Metrics and Alerts", and is a moniker to describe observability stacks.
```

Comment:
I thought it was `Logging, Monitoring and Alerting`?

Reply:
Gods below I hope not. Implying that “monitoring” is just about metrics is abhorrent.